api/invite_test.go

```diff
@@ -1866,60 +403,38 @@ func TestDismissMonitoringInvite(t *testing.T) {
 	}
 	mockTemplates, _ = templates.New(FAKE_CONFIG.I18nTemplatesPath, mockLocalizer)
 
-	patient := store.Patient{}
-	members := []store.Member{
-		{
-			UserID:           testing_token_hcp2,
-			TeamID:           "dismiss.team",
-			Role:             "admin",
-			InvitationStatus: "accepted",
-		},
-	}
-	notAdmin := []store.Member{
-		{
-			UserID:           testing_token_hcp2,
-			TeamID:           "dismiss.team",
-			Role:             "member",
-			InvitationStatus: "accepted",
-		},
-	}
-	team := store.Team{
-		Name:        "Dismiss team monitoring",
-		Description: "Dismiss team monitoring",
-		Members:     members,
-		ID:          "dismiss.team",
-	}
-	teamNotAdmin := store.Team{
-		Name:        "Dismiss team monitoring",
-		Description: "Dismiss team monitoring",
-		Members:     notAdmin,
-		ID:          "dismiss.team.not.admin",
-	}
-	mockPerms.SetMockNextCall(testing_token_uid1+"dismiss.team", &patient, nil)
-	mockPerms.SetMockNextCall(testing_token_hcp2+"dismiss.team", &team, nil)
-	mockPerms.SetMockNextCall(testing_token_hcp2+"dismiss.team.not.admin", &teamNotAdmin, nil)
-
-	mockPerms.On(
-		"UpdatePatientMonitoringWithContext", mock.Anything, mock.Anything, mock.Anything,
-	).Return(nil, nil)
-	mockPerms.On(
-		"GetPatientMonitoring", mock.Anything, mock.Anything, mock.Anything, mock.Anything,
-	).Return(&store.Patient{}, nil)
-
 	for idx, inviteTest := range inviteTests {
 		// don't run a test if it says to skip it
 		if inviteTest.skip {
 			continue
 		}
 		var testRtr = mux.NewRouter()
 
+		mockSeagull.SetMockNextCollectionCall("personToInvite@email.com"+"preferences", `{"Something":"anit no thing"}`, nil)
+		mockSeagull.SetMockNextCollectionCall(testing_uid1+"@email.org"+"preferences", `{"Something":"anit no thing"}`, nil)
+		mockSeagull.SetMockNextCollectionCall(testing_uid2+"@email.org"+"preferences", `{"Something":"anit no thing"}`, nil)
+		mockSeagull.SetMockNextCollectionCall(testing_uid2+"hcp@email.org"+"preferences", `{"Something":"anit no thing"}`, nil)
+		mockShoreline.On("TokenProvide").Return(testing_token)
+		mockAuth = auth.NewMock(false)
```

Comment:
would be something like 
```suggestion
		mockAuth = mockAuthNotAuthorized
```
or you use the mockAuthNotAuthorized variable instead of using mockAuth